ts/nni_manager/training_service/dlts/dltsClusterConfig.ts

```diff
@@ -1,14 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
Why remove dlts?

Reply:
DLTS has not been supported for long. It's config would not pass validation.
It's removed here because I don't want to fix lint.